src/e2e-test/java/io/cdap/cdap/ui/utils/Helper.java

```diff
@@ -299,6 +299,12 @@ public static void setShowWelcomeSessionStorage() {
       "window.sessionStorage.setItem('showWelcome','true');"));
   }
 
+  public static void setWindowPropertyCypress() {
+    JavascriptExecutor js = (JavascriptExecutor) SeleniumDriver.getDriver();
+    js.executeScript(String.format(
+            "window.Cypress = true"));
```

Comment:
In some places we look for `window.Cypress.cy.on`. We should probably set `window.Cypress to be an object with a no-op `cy.on` function.

Reply:
For now I've added Cypress object with `on` function